vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -438,14 +446,14 @@
   become: yes
   systemd:
     daemon_reload: yes
-  when: haproxy_service_name is changed
-
-- name: enable haproxy to connect to any port
-  become: yes
-  seboolean:
-    name: haproxy_connect_any
-    state: yes
-    persistent: yes
+  when: haproxy_serivce_name is changed
+
+#- name: enable haproxy to connect to any port
```

Comment:
why is this commented out?  Haproxy won't connect to local ports without this option.

Reply:
Ohh, I forgot to uncomment. It was remain commented out since on my environment the selinux is disabled and that part failed when your selinux is disabled. I'll fix it